games/bzflag/Portfile

```diff
@@ -18,8 +18,8 @@ homepage            https://www.bzflag.org/
 master_sites        https://download.bzflag.org/bzflag/source/${version}/
 use_bzip2           yes
 
-checksums           rmd160 6ea084cdcded82cc63dc9a0455bb631ad6454c56 \
-                    sha256 a8a13a58648798a6f6f969a2911cf21377fe45964f2edf46a327d1370ff1000d
+checksums           rmd160 4831f3915256fcb8c762acbd47bcc19afdaeb02c \
+                    sha256 09a46ec14cf956cb69fd069cee978de4dec321cb563036b16367ea8aae529bdc
```

Comment:
These days, we also have a `size` field that typically gets added after the `sha256` field. You can generate all these automatically with `port -v checksum`, fyi.

Reply:
Thank you! That's a convenient command to know. I was just using `openssl` 😅